.github/actions/bouf/Ensure-Location.ps1

```diff
@@ -0,0 +1,29 @@
+function Ensure-Location {
```

Comment:
Correct - in theory all the utility functions could be put into a single imported PowerShell script used just by the action (instead of the separate files used by the build script), but that's more a matter of style/taste.

Reply:
> Correct - in theory all the utility functions could be put into a single imported PowerShell script used just by the action (instead of the separate files used by the build script), but that's more a matter of style/taste.

Maybe some day it might make sense to turn the utility scripts into modules and install them as needed. Not a today problem.

If this particular method is okay to you, then I'll not block on it. We'll just need to be cognizant that there are multiple copies floating around if we have any changes to make.

Reply:
I suppose the files here could be symlinks rather than direct copies, but not sure how GitHub would handle that for actions.

Reply:
It's probably fine as-is for now. Just need to remember to port any fixes.